mkdocs.yml

```diff
@@ -533,6 +539,7 @@ nav:
       - Release Checklist: "development/core/release-checklist.md"
   - Release Notes:
     - "release-notes/index.md"
+    - Version 2.2: "release-notes/version-2.2.md"
```

Comment:
Would you mind also updating the `tool.towncrier` section in `pyproject.toml` to point to this file instead of `version-2.1.md`?